src/nl/hannahsten/texifyidea/folding/LatexSectionFoldingBuilder.kt

```diff
@@ -22,7 +22,7 @@ import nl.hannahsten.texifyidea.util.previousSiblingIgnoreWhitespace
 open class LatexSectionFoldingBuilder : FoldingBuilderEx() {
 
     companion object {
-        private val sectionCommandNames = arrayOf(
+        val sectionCommandNames = arrayOf(
```

Comment:
```suggestion
        private val sectionCommandNames = arrayOf(
```
No need to access this information from an implementation specific class. The commands are currently publicly defined in `Magic.Command.sectionMarkers`.